tests/data/reco_pixel_pkls/get_toclient_msg_pkl.py

```diff
@@ -27,8 +27,16 @@ def main():
     with open(args.pframe_pkl, "rb") as f:
         pframe = pickle.load(f)
 
-    with open(args.pframe_pkl.parent / "in.pkl", "wb") as f:
-        pickle.dump({"pframe": pframe, "reco_algo": args.reco_algo}, f)
+    # When extracting the debug .pkl from ewms-pilot, the in- pickles already contain the full message.
+    # Do we need to support "bare" pframes pickles at all?
+    # For the moment, this is a workaround.
+    if "pframe" in pframe:
+        # Effectively this is equivalent to copying the file.
+        with open(args.pframe_pkl.parent / "in.pkl", "wb") as f:
+            pickle.dump(pframe, f)
+    else:
+        with open(args.pframe_pkl.parent / "in.pkl", "wb") as f:
+            pickle.dump({"pframe": pframe, "reco_algo": args.reco_algo}, f)
```

Comment:
You're right. I believe the `milipede_original` test pickles are just pframes -- I don't remember how that came to be. But, I think it's fine since the data is okay. So we can leave the workaround. Can you add a comment explaining why?

Reply:
See issue #196 for this. I'd say replacing thest pickles for `millipede_original` and making all the test data uniform should not cost a lot of time, and will make this simpler.

Reply:
I hadn't seen #196. Those are all good points -- feel free to take those on. I'm indifferent whether those updates go into this PR or a separate one